tests/manage/pv_services/pvc_snapshot/test_small_files_pvc_cloned_status.py

```diff
@@ -0,0 +1,123 @@
+import logging
+import os
+import urllib.request
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.framework.testlib import (
+    E2ETest,
+    tier1,
+    polarion_id,
+    bugzilla,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@polarion_id("OCS-3946")
+class TestPvcClonedStatusOfSmallSizeFiles(E2ETest):
+    """
+    Tests to verify restored PVC status bound or not if
+    files size are smaller.
+    """
+
+    pvc_size = 1
+    pod_path = "/mnt"
+
+    @pytest.fixture()
+    def pvc(self, pvc_factory_class):
+        # Create a RWX Cephfs PVC
+        self.pvc_obj = pvc_factory_class(
+            interface=constants.CEPHFILESYSTEM,
+            access_mode=constants.ACCESS_MODE_RWX,
+            size=self.pvc_size,
+        )
+
+    @pytest.fixture()
+    def pod(self, pod_factory_class):
+        self.pod_obj = pod_factory_class(
+            pvc=self.pvc_obj,
+            status=constants.STATUS_RUNNING,
+            pod_dict_path=constants.PERF_POD_YAML,
+        )
+
+    @pytest.fixture()
+    def copy_files(self):
+        kernel_url = "https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-4.4.tar.xz"
```

Comment:
The file name "linux-4.4.tar.xz" appears 3 times here.
IMO you should set it as a variable in case you want to change the file in the future.